recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
+{% set compress_type = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "d9540e9af445b309e5eb89f4d6666d9f1ad755bb69c7cde3d7f814cbcbbf9144" %}
+{% set build_num = "0"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34
+    - matplotlib >=1.5.3
+    - numpy >=1.11.2
+    - six >=1.9.0
+  run:
+    - python
+    - enum34
+    - matplotlib >=1.5.3
+    - numpy >=1.11.2
+    - six >=1.9.0
+
+test:
+  # Python imports
+  imports:
+    - bezier
+    - functional_tests
+    - tests
+
+about:
+  home: https://github.com/dhermes/bezier
+  license: Apache 2.0
+  license_family: Apache
+  # license_file: LICENSE not in MANIFEST.in
```

Comment:
For `Apache 2.0` I believe you should follow the following:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually